modules/new_serverpod_auth/serverpod_auth_core/serverpod_auth_core_server/lib/src/session/util/authentication_info_extension.dart

```diff
@@ -7,6 +7,6 @@ extension AuthenticationInfoAuthSessionId on AuthenticationInfo {
   /// Assumes that the `authId` is set and the system uses `Uuid` auth session
   /// IDs, otherwise throws.
```

Comment:
Since `authId` is not nullable anymore, I think this doc comment should be updated

```suggestion
 /// Assumes that the system uses `Uuid` auth session IDs, otherwise throws.
```